.github/workflows/container-build.yml

```diff
@@ -61,23 +67,5 @@ jobs:
           target: booster-bitswap-dev
           push: ${{ github.event_name != 'pull_request' }}
           build-args: |
-            LOTUS_TEST_IMAGE=${{ env.LOTUS_TEST_IMAGE }}
-            FFI_BUILD_FROM_SOURCE=${{ env.FFI_BUILD_FROM_SOURCE }}
-      - name: Build lotus-dev
-        uses: ./.github/actions/container-builder
-        with:
-          name: lotus-dev
-          context: ./docker/devnet/lotus
-          push: ${{ github.event_name != 'pull_request' }}
-          build-args: |
-            LOTUS_TEST_IMAGE=${{ env.LOTUS_TEST_IMAGE }}
+            LOTUS_TEST_IMAGE=${{ env.LOTUS_SOURCE_IMAGE }}-${{ env.LOTUS_VERSION }}-${{ env.NETWORK_NAME }}
             FFI_BUILD_FROM_SOURCE=${{ env.FFI_BUILD_FROM_SOURCE }}
-      - name: Build lotus-miner-dev
-        uses: ./.github/actions/container-builder
-        with:
-          name: lotus-miner-dev
-          context: ./docker/devnet/lotus-miner
-          push: ${{ github.event_name != 'pull_request' }}
-          build-args: |
-            LOTUS_TEST_IMAGE=${{ env.LOTUS_TEST_IMAGE }}
-            FFI_BUILD_FROM_SOURCE=${{ env.FFI_BUILD_FROM_SOURCE }}
```

Comment:
That should be taken care of with `docker/all` `make` command. It will build the lotus and miner images within 20 seconds with the shipyard image as the base. Part of this build will be to inject entry points.

Reply:
Lotus source (Github repo - manual build) should work the same as before. I don't want to use the Lotus docker repo.

Reply:
right you are thank you